config/config.local.sample.neon

```diff
@@ -35,6 +35,11 @@ parameters:
 		sqlmode: TRADITIONAL
 		profiler: on
 
+	s3:
+		enabled: true # disable on dev, enable on production
```

Comment:
To jsem zvažoval, ale důležité pro mě je, že na produkci to musí být zapnuté. Protože to by default fungovat nebude, tak si to každý přepne. Disk by na první pohled „fungoval“ i na produkci.